_includes/runtime-generative.mdx

```diff
@@ -0,0 +1,5 @@
+:::tip
+
+You can [specify the generative model integration at query time](/developers/weaviate/search/generative#configure-a-generative-model-provider) without updating it in the collection configuration.
```

Comment:
Wdyt about `You can [override the generative integration settings at query time]...`

Reply:
Sounds better an technically more correct 👍